README.md

```diff
@@ -378,7 +378,7 @@ method takes one argument:
 
 1. `b` must be an Apply of a function
 
-    1. If `b` does not represent a function, the behaviour of `ap` is
+    1. If `b` does not represent an Apply of a function, the behaviour of `ap` is
        unspecified.
```

Comment:
I don't know why we're using _represent_ here. This would be clearer to me:

_If `b` is not an Apply of a function, the behaviour of `ap` is unspecified._